### PR TITLE
Feature 11:added first cypress test

### PR DESCRIPTION
**Pull request by DmByK**

### CODE REVIEW

**Comment by daniel-va** [comment]
For the future, doesn't need to fixed here: The correct naming rule for issue branches is `feature/asset-{issue-id}-{issue-title}`.

**Comment by daniel-va** [comment]
Personally, I would move the E2E-tests to `/e2e` rather than in `/cypress-e2e`. This keeps the project root somewhat cleaner.

File: cypress-e2e/.editorconfig
Comment (daniel-va) [inline cypress-e2e/.editorconfig:1]: Just a heads up that as soon as #122 is merged, the E2E-tests should use the same code style & linting rules as the rest of the project. We can leave the config here for now, but it should be removed when #122 goes live.

File: cypress-e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+1. Copy file cypress.env.json.example and rename to cypress.env.json
+2. Add username and password env variables to cypress.env.json (copy from cypress.config.ts)
+3. Add user credentials to cypress.env.json
+
+
+`cypress run --browser [browser of your choice e.g. firefox]`
+
+or if you prefer to use the cypress gui you can use `cypress open`.
+
```
Comment (daniel-va) [inline cypress-e2e/README.md:23]: Use multiline markdown with code highlighting:
````suggestion
```bash
cypress run --browser {browser of your choice e.g. firefox}
```
or if you prefer to use the cypress gui:
```bash
cypress open
```
````

File: cypress-e2e/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gin-x-cypress-e2e-tests",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "prepare": "husky install",
```
Comment (daniel-va) [inline cypress-e2e/package.json:6]: `npm install` fails for me with:
```
husky - .git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
```

File: cypress-e2e/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gin-x-cypress-e2e-tests",
```
Comment (daniel-va) [inline cypress-e2e/package.json:2]: This should probably be a different name, e.g `swissgeol-assets-e2e-tests`.

File: cypress-e2e/cypress/support/config/config.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+interface Config {
+  assetsBaseUrl: string;
+  basic_auth_username: string;
+  basic_auth_password: string;
```
Comment (daniel-va) [inline cypress-e2e/cypress/support/config/config.ts:4]: Rewrite these in camelCase if possible.

File: cypress-e2e/cypress/e2e/search.feature [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Feature: search
+
+  Background:
+    Given The user is logged in
+
+  Scenario: Save for test information
+    When A user types information into the search field
+    And The user clicks enter
+    Then The user should see a test information in resutls
```
Comment (daniel-va) [inline cypress-e2e/cypress/e2e/search.feature:9]: Fix typo.
```suggestion
    Then The user should see a test information in results
```

File: cypress-e2e/.editorconfig
Comment (DmByK) [reply cypress-e2e/.editorconfig:1]: Removed

**Review by daniel-va — APPROVED** [review]

**Pull request merged**